notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -2711,6 +2711,54 @@ the legal professions [Viktor's lawyer encouraged him to plead not guilty by rea
 criminal fraud [While being interrogated, Viktor Taransky offered to confess to fraud for his clandestine use of a virtual actress in his hit films.]
 
 
+movie: Scorcher (2002)
+======================
+
+:: Title
+Scorcher
+
+:: Description
+Scorcher is a 2002 science-fiction disaster film directed by James Seale and
+starring Mark Dacascos, John Rhys-Davies, Jeffrey Johnson, Tamara Davies,
+Mark Rolston, G.W. Bailey, Thomas F. Duffy, and Rutger Hauer.
+
+Synopsis: A group of scientists discover, after a disastrous nuclear accident,
+that the Earth's tectonic plates are shifting and creating immense pressure
+that will destroy the Earth in a fiery global eruption, and a few top
+scientists must find a way to stop it.
+
+:: Date
+2002-10-04
+
+:: References
+https://en.wikipedia.org/wiki/Scorcher_(film)
+
+:: Ratings
+1 <paul>
+
+:: Choice Themes
+volcanic cataclysm [The story is all about preventing a looming volcanic cataclysm of, quite literally, Earth shattering proportions.]
+
+:: Major Themes
+father and daughter [The troubled relationship between Julie her father Dr. Sallin was much featured, and was the similarly troubled relationship between Colonel Beckett and his teenage daughter Faith.]
+the military professions [Colonel Beckett led a team of military men, along with two scientists, on a mission to detonate two nuclear bombs in an abandoned Los Angeles.]
+neglectful parent [Julia held a grudge against her illustrious scientist father Dr. Sallin because he hadn't been there for her in her childhood. Colonel Beckett had a troubled relationship with his teenage daughter on account that he was always away because of his work.]
+rivalry [Julie scientist father Dr. Sallin were in a bitter feud.]
+nuclear weapons [Nuclear weapons were both the cause of and solution to the Pacific plate moving out of place, leaving the Earth's core exposed.]
+
+:: Minor Themes
+the dangers of unfettered scientific advancement [The Chinese inadvertently exposed the Earth's core during a nuclear test with potentially catastrophic consequences for life on the planet. This was mentioned at the start of the story but not followed up on in any way.]
+out of control scientific experiment [The Chinese inadvertently exposed the Earth's core during a nuclear test with potentially catastrophic consequences for life on the planet. This was mentioned at the start of the story but not followed up on in any way.]
+coping with the death of someone [Colonel Beckett grieved for a fellow soldier who'd died in the line of duty.]
+mother and daughter [Faith and her mother Tara tried to escape from Los Angeles together.]
+husband and wife []
```

Comment:
Oops, he did it again...

Reply:
Boy do I have egg on my face. Dang. shiny new motivation added.